shared/fs/header/container.native.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as Types from '../../constants/types/fs'
+import {navigateUp} from '../../actions/route-tree'
+import {compose, connect, setDisplayName, type Dispatch} from '../../util/container'
+import Header from './header.native'
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  onBack: () => dispatch(navigateUp()), // TODO: put if on route ...
+})
+
+const mergeProps = (stateProps, {onBack}, {path}) => {
+  const elems = Types.getPathElements(path)
+  return {
+    path,
+    title: elems.length > 1 ? elems.pop() : 'Keybase Files',
```

Comment:
don't think you want to do `pop`, as that modifies `elems`. I think you want `elems.length > 1 ? elems[elems.length - 1] : ...`.

Also, did you mean `elems.length > 0`?

Reply:
I meant to do `> 1`, since when it only has one element, it's `/keybase` and we'd be showing "Keybase Files". Will swap `pop()` out as that's probably cheaper. Thanks!